blocks/library/latest-posts/index.php

```diff
@@ -13,13 +13,20 @@
  * @return string Returns the post content with latest posts added.
  */
 function render_block_core_latest_posts( $attributes ) {
-	$recent_posts = wp_get_recent_posts( array(
-		'numberposts' => $attributes['postsToShow'],
-		'post_status' => 'publish',
-		'order'       => $attributes['order'],
-		'orderby'     => $attributes['orderBy'],
-		'category'    => $attributes['categories'],
-	) );
+	/** This filter is documented in wp-includes/widgets/class-wp-widget-recent-posts.php of WordPress core codebase */
+	$recent_posts = wp_get_recent_posts(
+		apply_filters(
+			'widget_posts_args',
+			array(
+				'numberposts' => $attributes['postsToShow'],
+				'post_status' => 'publish',
+				'order'       => $attributes['order'],
+				'orderby'     => $attributes['orderBy'],
+				'category'    => $attributes['categories'],
+			),
+			get_post()
```

Comment:
`get_post()` is not right, if you look at the hook docs at https://developer.wordpress.org/reference/hooks/widget_posts_args/ 

I suggest taking the array and storing it in a `$args` variable and then just duplicate it to ensure the same filter args are present:

```php
$args = array(
    /* ... */
);
/** This filter is documented in wp-includes/widgets/class-wp-widget-recent-posts.php */
$args = apply_filters( 'widget_posts_args', $args, $args );
$recent_posts = wp_get_recent_posts( $args );
```

Reply:
I'm not aware of what `$instance` is supposed to carry, but would an empty array not be less risky than sending values that may or may not collide with what may be matched?